### DISCUSSION TITLE
Private and public usernames and profile pictures

### DISCUSSION THREAD

**Discussion by eNkKVS4HxHc9jaq** [element-hq/element-meta#927] [Ideas]
#### What would you like to do?
Add the possibility to set a profile picture only seem by certain people, and the same for the username.
#### Why would you like to do it?
Adds additional control in terms of privacy to the app and to the users themselves.
### Have you considered any alternatives?

Currently impossible due to all the information being visible to everybody.

**Comment by daniellekirkwood** [comment]
Interesting idea - I've moved it to discussion for now as that's our process. 
Please keep leaving comments here, we'll take a look into it and move it back to an Issue when/if we're ready to build :)
